deis-workflow-manager/manifests/deis-workflow-manager-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  type: LoadBalancer
+  selector:
+    app: deis-workflow-manager
+  ports:
+    - name: https
+      port: 8443
+      targetPort: 8443
```

Comment:
It is listening via HTTPS, yes. It exposes routes via golang's standard `http.ListenAndServeTLS`. It would be easy enough to expose 443 from the container, but I chose a port in the non-privileged range to more easily expose a unique port on the node itself (i.e., a port that wouldn't be already active on another k8s service in the same cluster).